fix(compose): Make layout more responsive

kvviecien opened the pull request:
Better responsiveness and more organized layout for mobile

tadzik commented on src/styles.scss:
```diff
@@ -856,7 +843,7 @@ compose input.mat-chip-input {
 
 compose .mat-standard-chip {
-    bottom: 6px !important;
+    // bottom: 6px !important;
```
I think this commented-out bit can go now :)

kvviecien replied:
Right, I was probably checking something out and forgot about it as always.

kvviecien commented:
In my opinion, the Send button being bigger looks silly right next to the other ones being different size. Like someone made a mistake styling it.
If it's necessary to make it stand out, I'd rather make it a different color. Maybe grey out the other ones, take off a little bit of their saturation, and leave the Send button like it is now.

kvviecien commented:
Reordered the buttons and moved HTML and Attach buttons closer to the textarea.

I noticed the attachments looked a little wonky on mobile like this \/ so I fixed them too!
![Screenshot 2020-06-25 at 23 08 06](https://user-images.githubusercontent.com/24294620/85795607-dd797e80-b738-11ea-9642-646adcbacbe6.jpg)

Result:
![Screenshot 2020-06-25 at 23 04 04](https://user-images.githubusercontent.com/24294620/85795220-3bf22d00-b738-11ea-81fb-3b2f0503d21b.jpg)

kvviecien commented:
@gtandersen I thought putting Send as last would be better since it's best accessible when close to the edge of the screen, so maybe adding a little more space would be better if you think the buttons are too close.

gtandersen commented:
@kvviecien Alright, let's try that. If it doesn't help we can always make it a bit larger than the other buttons. ;)
